raw_data/nist_data/build_periodic_table.py

```diff
@@ -5,10 +5,8 @@
 import datetime
 import json
 import re
-from decimal import Decimal
 
 import requests
-from yapf.yapflib.yapf_api import FormatCode
```

Comment:
This can be commented, but best to have it available (at least in comments) to match formatting if the FormatCode line at end of file is revived. Sorry, GH won't let me add it back as suggestion.